### ISSUE TITLE
(Most Wanted) missing buttons in Video Menu

### ISSUE THREAD

**Issue by JOHN30011887** [xan1242/NFS-XtendedInput#14]
Iv noticed when compared to Pc controls and also with Widescreen Mod that this version is missing all the buttons when using a controller on the Video tab on the Main Menu, and when in the Video Menu ingame only the X button for advanced is missing.

Some screen to show what i mean
pc controls
![1](https://user-images.githubusercontent.com/12979984/166118166-4e07b2ef-d83e-4059-860f-698afff69b13.png)
xbox controls
![2](https://user-images.githubusercontent.com/12979984/166118167-081f2c54-0114-447d-98c5-34b5428d9af8.png)
pc controls
![3](https://user-images.githubusercontent.com/12979984/166118169-8b64a800-213e-4c20-adf9-9163d3b21250.png)
xbox controls
![4](https://user-images.githubusercontent.com/12979984/166118171-5fc24834-7465-4634-9f1c-ea0bfcbe327c.png)

**Comment by HarGabt** [comment]
It's unlikely to be caused by mod, cause the thing mod does in this case is calling console counterparts of frontend, which, logically, don't have any graphics settings.

**Comment by JOHN30011887** [comment]
Just noticed also Carbon has the issue but its just the X button for Advanced thats missing from both Video menus, the main menu one and ingame one.

**Comment by JOHN30011887** [comment]
@HarGabt oh, didnt think of it that way, as you say consoles dont have that advanced button option, dont know if its possible to add the option in

**Comment by HarGabt** [comment]
> if its possible to add the option in
Only by editing .fng files that contain these buttons and actions. 
But as for now, there isn't any software in complete state to do this yet.
https://github.com/NFSTools/FEngLib

**Comment by xan1242** [comment]
> It's unlikely to be caused by mod, cause the thing mod does in this case is calling console counterparts of frontend, which, logically, don't have any graphics settings.

You got it correct. It would technically be possible to add them by editing the package and adding the object groups in.

No public tools exist, there is that FEngLib you mentioned, I worked on some of my own tools too. Sooner or later, once it's possible to add it in, you'll be able to see them then (and subsequently redirect it to the new package).

Right now I could work around it by excluding that screen from the switching but I do not feel like it is a very important issue at the moment.

It'll potentially have to be done anyway, considering the Controls settings menu has a similar issue in other games and I am planning to use that menu for configuration as well in the future.

**Comment by JOHN30011887** [comment]
Yeah this issue aint important so i would say its best to leave the menu how you have done it so far, once video settings are adjusted you dont need to really go back in there anyway.

I look forward to see what else you do to these classics :)

**Comment by AvinashReddy3108** [comment]
> > It's unlikely to be caused by mod, cause the thing mod does in this case is calling console counterparts of frontend, which, logically, don't have any graphics settings.
> 
> You got it correct. It would technically be possible to add them by editing the package and adding the object groups in.
> 
> No public tools exist, there is that FEngLib you mentioned, I worked on some of my own tools too. Sooner or later, once it's possible to add it in, you'll be able to see them then (and subsequently redirect it to the new package).
> 
> Right now I could work around it by excluding that screen from the switching but I do not feel like it is a very important issue at the moment.
> 
> It'll potentially have to be done anyway, considering the Controls settings menu has a similar issue in other games and I am planning to use that menu for configuration as well in the future.

A good middle ground would be to just replace the button textures in said menus rather than bringing the whole console counterpart of them?
